lib/extractor/index.ts

```diff
@@ -196,7 +196,12 @@ function layersWithLatestFileModifications(
 
       // not adding the original file to extractedLayers
       // and removing it from the set since it can be found in consecutive layers
-      if (removedFilesToIgnore.has(filename)) {
+      if (
+        removedFilesToIgnore.has(filename) ||
+        Array.from(removedFilesToIgnore).some((removedFile) =>
+          filename.startsWith(removedFile),
+        )
```

Comment:
Should we still try to remove the `filename` from the `removedFilesToIgnore` set in case the directory was removed and not the file (basically if the if condition is `true` because of the second part)?

Reply:
The point was "if the file was in a directory that was removed, remove that file". I do see an issue now though, where where if the folder was created again in the next layer, it won't be registered

Reply:
The issue that I thought about turns out is handled by the fact that we process the layers from bottom to top. I added a test to showcase this